src/libopensc/reader-pcsc.c

```diff
@@ -1132,6 +1136,13 @@ static void detect_reader_features(sc_reader_t *reader, SCARDHANDLE card_handle)
 		}
 	}
 
+	for (i = 0; i < sizeof(broken_readers)/sizeof(broken_readers[0]); ++i) {
```

Comment:
Maybe add a "XXX" kind of header here to remind that this should at some point, be removed ?